x-pack/plugins/apm/server/routes/security.ts

```diff
@@ -12,6 +12,9 @@ export const indicesPrivilegesRoute = createRoute(() => ({
   path: '/api/apm/security/indices_privileges',
   handler: async ({ context, request }) => {
     const setup = await setupRequest(context, request);
-    return getIndicesPrivileges(setup, context.logger);
+    return getIndicesPrivileges(
+      setup,
+      context.security?.license.isEnabled() ?? false
```

Comment:
Mind changing `getIndicesPrivileges` to a parameters to destructured object? Will make it easier to understand the boolean param.